.github/workflows/validate-helm-chart.yml

```diff
@@ -50,3 +53,31 @@ jobs:
             echo ""
             echo "Helm values documentation is correct!"
           fi
+
+  helm-generation-check:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      
+      - name: Generate current YAML
+        run: |
+          helm template helm/chart > current.yaml
+      
+      - name: Generate new Helm Chart
+        run: make helm-package
```

Comment:
make helm package downloads via github api both helmify and kustomize, should we add a cache here too? *stares_at_pr_with_caching_obsessed_eyes*

Reply:
good catch! let's add the caching step as it is done here:
https://github.com/keptn/lifecycle-toolkit/blob/main/.github/workflows/CI.yaml#L134-L139
@rakshitgondwal

Reply:
what `path` and `key` should I include in this step?

Reply:
same path, same key :)